src/System.Collections.NonGeneric/src/System/Collections/Comparer.cs

```diff
@@ -34,6 +43,36 @@ public Comparer(CultureInfo culture)
             _compareInfo = culture.CompareInfo;
         }
 
+#if netstandard17
```

Comment:
We generally prefer splitting these tests into another file instead of #if def'ing them.